packages/astro/src/core/app/index.ts

```diff
@@ -41,6 +41,7 @@ export interface RenderErrorOptions {
 	routeData?: RouteData;
 	response?: Response;
 	status: 404 | 500;
+	runMiddleware?: boolean;
```

Comment:
I would keep it not optional, otherwise we have tree values: `undefined`, `true` and `false`

Reply:
Can we document this option and explain what it is?

Reply:
I think there really are 3 options: run middleware, prevent middleware, and default.

Reply:
Great! One more reason to document and explain what the three options do :)

Reply:
yeah, good point